Examples/Tests/electrostatic_sphere_eb/analysis_rz_mr.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python
+
+# Copyright 2024 Olga Shapoval, Edoardo Zoni
+#
+# This file is part of WarpX.
+#
+# License: BSD-3-Clause-LBNL
+
+# This script tests the embedded boundary in RZ.
+# A cylindrical surface (r=0.1) has a fixed potential 1 V.
+# The outer surface has 0 V fixed.
+# Thus the analytical solution has the form:
+# phi(r) = A+B*log(r), Er(r) = -B/r.
+
+# Possible errors:
+# tolerance: 0.004
+# Possible running time: < 1 s
+
+import os
+import sys
+
+import numpy as np
+from openpmd_viewer import OpenPMDTimeSeries
+
+sys.path.insert(1, '../../../../warpx/Regression/Checksum/')
+import checksumAPI
+
+tolerance = 0.0041
+
+fn = sys.argv[1]
+
+def find_first_non_zero_from_bottom_left(matrix):
+    nj = len(matrix[0,:])
+    ni = len(matrix[:,0])
+    for i in range(ni):
+        for j in range(nj):
+            if (matrix[i][j] != 0) and (matrix[i][j] != np.nan):
+                return (i, j)
+    return i, j
+
+def find_first_non_zero_from_upper_right(matrix):
+    nj = len(matrix[0,:])
+    ni = len(matrix[:,0])
+
+    for i in range(ni-1, -1, -1):
+        for j in range(nj-1, -1, -1):
+            if (matrix[i][j] != 0) and (matrix[i][j] != np.nan):
+                return (i, j)
+    return i,j
```

Comment:
@prkkumar 
Based on your work with mesh refinement simulations in WarpX, do you have experience with finding the min/max coordinates of a mesh refinement patch starting from the data available through an openPMD time series? I see that it is done manually here (index search where a given field has a valid value), but I wonder if there is a built-in way to do this.